src/runners/ingest_runner.py

```diff
@@ -10,14 +10,14 @@
 # The pipeline runner also has to iterate through all the connection table and call ingest() on them.
 
 def main(connection_table="%_CONNECTION"):
-    # for name in os.listdir('../ingestion'):
-    #     log.info(f"invoking {name}")
-    #     try:
-    #         res = subprocess.call(f"python ../ingestion/{name}", shell=True)
-    #         log.info("subprocess returns: ", res)
-    #         log.info(f"{name} invoked")
-    #     except Exception as e:
-    #         log.error(f"failed to run {name}", e)
+    for name in os.listdir('../ingestion'):
```

Comment:
this change is not related to the Azure connector and duplicates one on the branch you are merging into -- could you try rebasing and squashing all these changes into a single diff?

Reply:
rebased

Reply:
Are you sure? Github UI says there is still no clean merge
> This branch cannot be rebased due to conflicts
> Rebasing the commits of this branch on top of the base branch cannot be performed automatically due to conflicts encountered while reapplying the individual commits from the head branch.

Reply:
hmm... Maybe? I just pushed, did that fix it?